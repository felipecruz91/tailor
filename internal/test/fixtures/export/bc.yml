apiVersion: v1
kind: List
items:
- apiVersion: build.openshift.io/v1
  kind: BuildConfig
  metadata:
    name: bar
  spec:
    nodeSelector: null
    output:
      to:
        kind: ImageStreamTag
        name: bar:latest
    postCommit: {}
    resources: {}
    runPolicy: Serial
    source:
      git:
        ref: master
        uri: https://github.com/foo/bar.git
      type: Git
    strategy:
      dockerStrategy:
        from:
          kind: ImageStreamTag
          name: base:latest
          namespace: foo-dev
      type: Docker
    triggers: []
