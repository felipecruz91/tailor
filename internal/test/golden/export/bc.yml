apiVersion: template.openshift.io/v1
kind: Template
objects:
- apiVersion: build.openshift.io/v1
  kind: BuildConfig
  metadata:
    name: bar
  spec:
    nodeSelector: null
    output:
      to:
        kind: ImageStreamTag
        name: bar:latest
    postCommit: {}
    resources: {}
    runPolicy: Serial
    source:
      git:
        ref: master
        uri: https://github.com/foo/bar.git
      type: Git
    strategy:
      dockerStrategy:
        from:
          kind: ImageStreamTag
          name: base:latest
          namespace: ${TAILOR_NAMESPACE}
      type: Docker
    triggers: []
parameters:
- name: TAILOR_NAMESPACE
  required: true
